pkg/operator/starter.go

```diff
@@ -29,11 +29,7 @@ const (
 	operatorVersionEnvName = "OPERATOR_IMAGE_VERSION"
 )
 
-var targetDeploymentNames = sets.NewString(
-	api.SignerControllerDeploymentName,
-	api.APIServiceInjectorDeploymentName,
-	api.ConfigMapInjectorDeploymentName,
-)
+var targetDeploymentNames = sets.NewString(api.ServiceCADeploymentName)
```

Comment:
that's a big set.

Reply:
Room for cleanup around status, for sure, but I'd rather tackle that separately. The main intent of this PR is reducing the resource footprint of the operator in advance of the bundle injector refactor to avoid unnecessary churn in resources managed by the operator.